app/script.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+// sends a GET request to our server
+$(function() {
+  $("#name").click(function() {
+    // first argument is the route
+    $.get('adjective', function(response) {
+      console.log(response);
```

Comment:
Once you have your code working, make a new branch and take out extraneous console logs (like these). You'll use that branch for production (e.g. pushing to Heroku) and turning in for assignments.